docs/node/admin/kwil-admin/snapshot/create.mdx

```diff
@@ -0,0 +1,59 @@
+---
+sidebar_position: 1
+sidebar_label: "create"
+id: "kwil-admin-snapshot-create"
+title: "kwil-admin snapshot create"
+description: "Creates a snapshot of the database."
+slug: /ref/kwil-admin/snapshot/create
+---
+
+## kwil-admin snapshot create
+
+Creates a snapshot of the database.
+
+### Synopsis
+
+
+This command creates a logical snapshot of the database and genesis file to initialize a new network. The genesis file is created with the app_hash set to the hash of the snapshot(uncompressed) and the genesis validators set to the validators from the previous network. This command interacts directly with the underlying PostgreSQL server, bypassing any interactions with the kwild node. It requires user with administrative privileges on the database to create snapshot without any restrictions. 
+	
+This command is intended to be used for network migrations to generate exportable snapshots used for initializing a new network, serving as the genesis state. The snapshots generated by this command are different from the ones generated by the snapshot store on running the Kwild instance. The snapshots generated by the snapshot store are used for state sync and are not meant to be used for initializing a new network as they contains state which shouldn't exist at genesis.
```

Comment:
I feel like this paragraph almost might be better before the other one.  But it's a little repetitive.  Probably ok, but I'm on the fence.

---

`Kwild` => `kwild`.  We should be as consistent as possible in our docs.

---


When saying "The snapshots generated by the snapshot store"  I don't think the reader has an idea what the snapshot store is.  I think we can just say "The snapshots generated by kwild during normal operation are different from the snapshots generated by this tool.  This tool is to prepare a new network based on the final state of an existing network, while kwild's snapshots support fast "state sync" for nodes joining an existing network."